ui/apps/platform/src/Containers/Clusters/InitBundles/InitBundlesHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactElement } from 'react';
+import {
+    Breadcrumb,
+    BreadcrumbItem,
+    Flex,
+    FlexItem,
+    PageSection,
+    Text,
+    Title,
+} from '@patternfly/react-core';
+
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import PageTitle from 'Components/PageTitle';
+import { clustersBasePath, clustersInitBundlesPath } from 'routePaths';
+
+const titleInitBundles = 'Cluster init bundles';
+
+export type InitBundlesHeaderProps = {
+    alignRightElement?: ReactElement | null;
```

Comment:
If there is the potential for multiple buttons in the future, what do you think about `headerActions?: ReactElement[] | null;` as an alternative naming? That would abstract over the positioning and focus more on child component intent.

Reply:
Yes, indeed. What do you think about `ReactNode` versus `ReactElement[]` for arbitrary content?